olp-cpp-sdk-dataservice-write/test/unit/TestStreamLayerClient.cpp

```diff
@@ -112,6 +114,17 @@ void PublishSdiiSuccessAssertions(
   EXPECT_FALSE(result.GetResult().GetTraceID().GetGeneratedIDs().at(0).empty());
 }
 
+template <typename T>
+void PublishCancelledAssertions(
```

Comment:
What is PublishCancelledAssertions?
Do you mean CheckResponse?

Reply:
These assertions are shared across all write tests where publish requests are cancelled.

Reply:
The name of the function is wrong. What do you publish? I dont see any publish? What you do in the function is checking if the result is ok. So the function should be called CheckResponse()

Reply:
there are already functions named 
```
PublishDataSuccessAssertions()
PublishFailureAssertions()
```
Also similar names are used in tests for other layer types. I agree the naming is wrong but at least it is consistent.

Reply:
When moving you have the chance to change it.